docs/data/material/migration/migration-v6/v6-component-changes.md

```diff
@@ -0,0 +1,38 @@
+# Breaking changes in v6: core components
```

Comment:
Instead of page, can we please add readme file in the @mui/material-next package? Maybe with name `migration.md` next to readme. We can add info in the readme, that they should follow the `migraiton.md` guidelines.